### ISSUE TITLE
News From Date and Design Active From is set when setting Special Price for product.

### ISSUE THREAD

**Issue by thomasnordkvist** [magento/magento2#4387]
## Steps to reproduce

In backend, set a special price for a product.
## Expected result
1. That only the Special Price is set.
## Actual result
1. The Special Price is set for the product
2. If you leave the field Special Price From Date blank it is default set to todays date.(This i could consider ok)
3. The field for News From Date in Autosettings is set to todays date. It's possible to change but it's not possoble to clear it.
4. The filed Active From in Design is also set and can't be cleared.

Workaround for now is to set the News From Date and News To Date to passed dates if you are using for example a widget showing new products.

**Comment by pboisvert** [comment]
@antboiko can you check if this is working as intended/documented or a bug?

**Comment by antboiko** [comment]
@thomasnordkvist 
Hi Thomas, could you please tell me what Magento version you use?

Thanks,
Anton.

**Comment by thomasnordkvist** [comment]
I have this issue in version 2.0.4 whish is the only version I have access to at the moment

**Comment by antboiko** [comment]
@thomasnordkvist thanks for reporting this issue. Internal ticket MAGETWO-52577 was created.

**Comment by ghost** [comment]
Hi, 
Has this issue been addressed at all? I am using Magento 2.1.2 and still the same issue is happening.
Is there a release planned that fixes this?

**Comment by olysenko** [comment]
Hi @joe980 , I increased priority for the fix. It won't be delivered in the next release 2.1.3, so it scheduled for 2.1.4. If any changes happened I  inform you

**Comment by stephansteiner** [comment]
2.1.4 and 2.1.5 are out and bug still exists. can you raise priority so it will be included in 2.1.6?

**Comment by jarhody** [comment]
2.15 still an issue for me.

I used this work around for now. Simply adding a date to all products that have NULL values in these fields.

Login in to PhpMyAdmin
Go to table catalog_product_entity_datetime
Find the attribute ids of news from date and news to date (or go to the attribute in admin and find id in the url key)
Run query below **(change the atrribute_id(s) to match your ids obtained from above)**

UPDATE catalog_product_entity_datetime
SET  value = '2000-01-01 00:00:00'
WHERE attribute_id = 77 AND value IS NULL OR attribute_id = 78 AND value IS NULL

**Comment by mattcoz** [comment]
Still an issue in 2.1.6

**Comment by digitalredefined** [comment]
Confirmed still an issue for 2.1.6

**Comment by ghost** [comment]
Confirmed still an issue for 2.1.7

**Comment by cherreman** [comment]
The problem is that Magento\Catalog\Model\Attribute\Backend\Startdate sets the from-date to the current date when the product has a special price. This also incorrectly sets all other start/from dates, for instance the news_from_date. A possible fix is to add an extra check for the special_from_date attribute before setting it:

Change

```
if ($startDate == '' && $object->getSpecialPrice()) {
  $startDate = $this->_localeDate->date();
}
```

to

```
if ($attributeName == "special_from_date" && $startDate == '' && $object->getSpecialPrice()) {
  $startDate = $this->_localeDate->date();
}
```

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-72252

**Comment by magento-engcom-team** [comment]
@thomasnordkvist thank you for your report.
The fix for this issue is already fixed in 2.2.0 release
